### DISCUSSION TITLE
Passing another variable on Cloumn use view

### DISCUSSION THREAD

**Discussion by echoskydev** [rappasoft/laravel-livewire-tables#1315] [Q&A] [answered]
`
return [
      Column::make('ID')->searchable()->sortable(),
      Column::make('Brand Name', 'brand.name')->searchable()->sortable(),
      Column::make('Name')->searchable()->sortable(),
      Column::make('Color')->searchable()->sortable()->view('cars.table-color'),
      Column::make('Accidents', 'accidents_count')->sortable(),
      Column::make()->view('cars.table-actions'),
];

`

i need passing another variable on ->view('cars.table-color')

->view('cars.table-color', ['var' => 1]) ===> not working

**Comment by lrljoe** [comment, marked as answer]
You can access variables declared elsewhere in the component, from your blade with the standard
{{ $this->variable }}

Look at the format() approach if you want to pass multiple variables into a view

```
Column::make('Name')
    ->format(
        fn($value, $row, Column $column) => view('my.custom.view')->withValue($value)
    ),
```

**Reply by echoskydev** [reply]
Thank you for your help.
I using this solution it works!!!

```php
Column::make('Color')
 ->searchable()
 ->sortable()
 ->format(
      fn ($value, $row, Column $column) => '<a href="' . route('car.show', ['var' => $this->var, 'id' => $row->id]) . '">' . $value . '</a>'
   )
 ->html(),
```
